src/renderer/components/+namespaces/namespace-select.tsx

```diff
@@ -13,17 +13,12 @@ import { kubeWatchApi } from "../../api/kube-watch-api";
 
 interface Props extends SelectProps {
   showIcons?: boolean;
-  showClusterOption?: boolean; // show cluster option on the top (default: false)
-  clusterOptionLabel?: React.ReactNode; // label for cluster option (default: "Cluster")
```

Comment:
Not used in project.

Reply:
This is a breaking change

Reply:
It wasn't even documented (and used I guess).

Reply:
Oh I see, sorry. It is exposed but it hasn't been exposed in a release.

Reply:
Anyway, thanks for pointing it out. It seems that `showClusterOption` is used internally in `<AddRoleBindingDialog/>`, so this one is reverted.